packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -51,7 +51,7 @@ import {
 export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
 	public readonly protocolName = "fluid-odsp:";
 
-	private readonly nonPersistentCache = new NonPersistentCache();
+	public readonly nonPersistentCache: INonPersistentCache = new NonPersistentCache();
```

Comment:
public is not good. Maybe you need protected?

Reply:
I am now exposing just the snapshot cache through the getter. We won't be able to access in prefetch if we make it protected.